Update C++ documentations (#21925)
DataDog/documentation: content/en/tracing/trace_pipeline/ingestion_mechanisms.md

[1]: /tracing/trace_collection/dd_libraries/php
{{% /tab %}}
{{% tab "C++" %}}

> Starting in version version [v0.1.0][1], the Datadog C++ library supports the following configurations:

Reviewer: ```suggestion
Starting in [v0.1.0][1], the Datadog C++ library supports the following configurations:
```

@env DD_TRACE_SAMPLING_RULES=[{"service": "my-service", "sample_rate": 0.5}]
```

> C++ does not provide integrations for out-of-the-box instrumentation, but it's used by proxy tracing such as Envoy, Nginx, or Istio. Read more about how to configure sampling for proxies in [Tracing proxies][2].

Reviewer 2: ```suggestion
C++ does not provide integrations for automatic instrumentation, but it's used by proxy tracing such as Envoy, Nginx, or Istio. Read more about how to configure sampling for proxies in [Tracing proxies][2].
```
